sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Release History
 
-## 5.5.0-beta.1 (Unreleased)
+## 5.4.1 (2021-05-11)
 
+### Changes
+
+#### New Features
+
+- The processor will now perform validation of core configuration and permissions at startup, in order to attempt to detect unrecoverable problems more deterministically.  Validation is performed unobtrusively and will not delay claiming of partitions.  One important note is that validation should be considered point-in-time and best effort; it is not meant to replace monitoring error handler activity.
+
+- Partition initialization has been moved to a background operation.  This will allow partitions to be more efficiently managed and speed ownership claims, especially when using a Greedy load balancing strategy or when the processor is recovering from some error conditions.
```

Comment:
```suggestion
- Partition initialization has been moved to a background operation.  This will allow partitions to be more efficiently managed and speed up ownership claims, especially when using the `LoadBalancingStrategy.Greedy` configuration or when the processor is recovering from some error conditions.
```